src/elements/RadioPill/RadioPill.stories.tsx

```diff
@@ -0,0 +1,30 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import { RadioPill, RadioPillProps } from "./RadioPill";
+import IconIcons from "@elements/Icon/Generated/IconIcons";
+
+export default {
+    title: "Elements/RadioPill",
+    component: RadioPill,
+    args: {
+        label: "Label",
+        active: true,
+    },
+    argTypes: {
+        icon: {
+            options: ["None", "Icon"],
+            mapping: {
+                None: null,
+                Icon: <IconIcons />,
+            },
+            defaultValue: "None",
+            control: { type: "select" },
+        },
+    },
+} as Meta<RadioPillProps>;
+
+const Template: Story<RadioPillProps> = (args) => <RadioPill {...args} />;
```

Comment:
If there's only one story you could add `import RadioPill as RadioPillComponent` and export the story directly:
```suggestion
export const RadioPill: Story<RadioPillProps> = (args) => <RadioPillComponent {...args} />;
```

Reply:
Done